### ISSUE TITLE
E2E messaging can be integrated into an ActivityPub application's UI like Twitter does

### ISSUE THREAD

**Issue by jernst** [swicg/activitypub-e2ee#31]
"As the user of an ActivityPub application that looks like Twitter, in the application's UI I can switch back and forth between public and E2E messages as easily as I can between the public feed and DMs on Twitter."

This goes to seamless UX.

**Comment by benpate** [comment]
Yes. This would be best from the user's perspective and apps should try to make this as seamless as possible.

One thing I'm running into in my design is that the E2EE messaging is a fundamentally different application that's built (and runs) very differently from the rest of the system.  I can still tie the navigation into the rest of the app, but I can't easily display DMs side by side with E2EE messages.  I don't like it, but the best I've got at the moment would be a menu that looks something like this:

```
My Inbox
---------
Direct Messages
Encrypted Messages

Notifications

Inbox Folder 1
Inbox Folder 2
Inbox Folder 3
```

I'm going to keep looking for other UX options, and I'd love to collect others' suggestions.
